src/services/pig.py

```
@@ -202,14 +208,14 @@ async def update_pig(
         if body.is_rolling_admission is not None:
             pig.is_rolling_admission = body.is_rolling_admission
 
+        if body.websites is not None:
+            self._replace_websites(id, body.websites)
+
         try:
             self.pig_repository.update(pig)
         except IntegrityError:
             raise HTTPException(409, detail="기존 시그/피그와 중복된 항목이 있습니다")
```

Ruff (0.15.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)